braze/client.py

```diff
@@ -180,10 +180,8 @@ def user_export(self, external_ids=None, email=None, fields_to_export=None):
         elif email:
             payload["email"] = email
 
-        if not fields_to_export:
-            fields_to_export = []
-
-        payload["fields_to_export"] = fields_to_export
+        if fields_to_export:
+            payload["fields_to_export"] = fields_to_export
```

Comment:
NB: Something to double check/test: So the default behavior now will be to fetch all fields vs no user fields/no-op earlier.